### ISSUE TITLE
Add alt-text to avatar-images 

### ISSUE THREAD

**Issue by Angamanga** [ushahidi/platform#4103]
## Description
On the post-cards, there is a small image with the user-avatar. There is no alt-text for these images so we need to add that to improve accessibility.

## TODO:
Find all img-elements that are displaying avatars. HINT! We use gravatar to display the images so searching for that will get you in the right direction. Add an alt-tag to each of these with the text "user avatar". Note. Do NOT change the avatar in the mode-bar.html. That one is covered by another issue since it has other requirements.

Aha! Link: https://ushahiditeam.aha.io/features/PROD-855

**Comment by joneskj55** [comment]
Hi @Angamanga I'd like to take this one.

**Comment by Shazeen15** [comment]
@Angamanga 
I would like to work on this issue.

Shazeen Fabius
Outreachy Candidate

**Comment by Sunairaa** [comment]
Hey! @Angamanga can i work on this issue?

**Comment by treasurechristain** [comment]
@Angamanga I'm Amadikwa Joy N. An outreachy Applicant. Can I work on this issue please.

**Comment by pg00019** [comment]
Hii @Angamanga . I would like to work on this issue.

**Comment by babra99** [comment]
Hii @Angamanga i am Babra. I am outreachy  appilicant. Can i owrk on this issue?

**Comment by ulomau** [comment]
@Angamanga  please can i claim this issue

**Comment by bimbolabuari** [comment]
@rowasc @Angamanga,
Hi, I'm Abimbola an outreachy applicant and I would love to work on this issue

**Comment by Anisha-28** [comment]
@Angamanga i'm an outreachy applicant .. i can work on this issue

**Comment by Mod5ied** [comment]
@Angamanga I would really love to solve this issue and improve accessibility onsite.
I'm an Outreachy candidate.

**Comment by bimbolabuari** [comment]
@Angamanga @rowasc,
I would love to work on this issue

**Comment by codifyit** [comment]
Hi Can I work on this issue? I am an outreachy applicant

**Comment by Barkty** [comment]
@Angamanga I would really love to work on this issue. I'm an Outreachy applicant.

**Comment by Barkty** [comment]
@Angamanga I would really love to work on this issue. I'm an Outreachy applicant.

**Comment by Kennygrace** [comment]
@Angamanga I'm Kennygrace, an outreachy applicant, I would like to work on this issue

**Comment by Angamanga** [comment]
@joneskj55 Please go ahead!

**Comment by tuxpiper** [comment]
@all-contributors please add @bimbolabuari for code

**Comment by tuxpiper** [comment]
@all-contributors please add @bimbolabuari for code
